crimsobot/cogs/mystery.py

```diff
@@ -58,6 +58,88 @@ async def tarot(self, ctx: commands.Context, spread: str = 'ppf') -> None:
         fp, descriptions = tarot.reading(spread)
         await ctx.send('\n'.join(descriptions), file=discord.File(fp, 'reading.png'))
 
+    @commands.command(aliases=['tarotcard'])
+    async def card(self, ctx: commands.Context) -> None:
+        """Inspect a tarot card chosen from the menu for further study."""
+
+        # the suits
+        suit_dict = {0: 'Major arcana', 1: 'Wands', 2: 'Pentacles', 3: 'Cups', 4: 'Swords'}
+        suit_list = []
+        for suit in suit_dict:
+            suit_list.append('{}. {}'.format(suit, suit_dict[suit]))
```

Comment:
not really sure why this is a dict?
would recommend something like this:
```suggestion
        suits = ['Major arcana', 'Wands', 'Pentacles', 'Cups', 'Swords']
        suit_list = []
        for idx, suit in enumerate(suits):
            suit_list.append('{}. {}'.format(idx, suit))
```
keeping in mind you'll need to change any later references of `suit_dict` to `suits`